packages/tomls/src/omnibus/reya_cronos.toml

```diff
@@ -0,0 +1,241 @@
+passivePoolExchangeFeeCollectorOwner = "0xaE173a960084903b1d278Ff9E3A81DeD82275556"
+ranksBot = "0xaE173a960084903b1d278Ff9E3A81DeD82275556"
+
+[var.cannonClonePackages]
+coreProxyPackage = "reya-core:1.0.0@proxy"
+coreRouterPackage = "reya-core:1.0.1@router"
+coreAccountNftRouterPackage = "reya-core:1.0.1@account-nft-router"
+rUSDProxyPackage = "reya-rusd:1.0.0@proxy"
+rUSDRouterPackage = "reya-rusd:1.0.0@router"
+tokensProxyPackage = "reya-tokens:1.0.0@proxy"
+tokensRouterPackage = "reya-tokens:1.0.0@router"
+passivePoolProxyPackage = "reya-exchange-passive-pool:1.0.0@proxy"
+passivePoolRouterPackage = "reya-exchange-passive-pool:1.0.1@router"
+passivePerpProxyPackage = "reya-instrument-passive-perp:1.0.0@proxy"
+passivePerpRouterPackage = "reya-instrument-passive-perp:1.0.1@router"
+peripheryProxyPackage = "reya-periphery:1.0.0@proxy"
+peripheryRouterPackage = "reya-periphery:1.0.2@router"
+oracleManagerProxyPackage = "reya-oracle-manager:1.0.0@proxy"
+oracleManagerRouterPackage = "reya-oracle-manager:1.0.1@router"
+exchangePassProxyPackage = "reya-exchange-pass-nft:1.0.0@proxy"
+exchangePassRouterPackage = "reya-exchange-pass-nft:1.0.0@router"
+ranksProxyPackage = "reya-ranks:1.0.0@proxy"
+ranksRouterPackage = "reya-ranks:1.0.0@router"
+
+[var.account_nft_vars]
+accountNftId = "<%= formatBytes32String('accountNFT') %>"
+accountNftName = "Reya Account NFT"
+accountNftSymbol = "REYA"
+accountNftUri = "https://reya.xyz/"
+
+[var.exchange_pass_vars]
+exchangePassName = "Reya Exchange Pass"
+exchangePassSymbol = "ReyaEPass"
+exchangePassUri = "https://reya.xyz/"
+
+[var.initial_proxy_vars]
+usdcSalt = "usdc-main"
+usdtSalt = "usdt-main"
+daiSalt = "dai-main"
+wethSalt = "weth-main"
+wbtcSalt = "wbtc-main"
+
+[var.upgrade_proxy_vars]
+coreProxy = "<%= imports.reyaCoreProxy.contracts.InitialProxy.address %>"
+rUSDProxy = "<%= imports.reyaRusdProxy.contracts.InitialProxy.address %>"
+passivePoolProxy = "<%= imports.reyaPassivePoolProxy.contracts.InitialProxy.address %>"
+passivePerpProxy = "<%= imports.reyaPassivePerpProxy.contracts.InitialProxy.address %>"
+peripheryProxy = "<%= imports.reyaPeripheryProxy.contracts.InitialProxy.address %>"
+oracleManagerProxy = "<%= imports.reyaOracleManagerProxy.contracts.InitialProxy.address %>"
+exchangePassProxy = "<%= imports.reyaExchangePassProxy.contracts.InitialProxy.address %>"
+ranksProxy = "<%= imports.reyaRanksProxy.contracts.InitialProxy.address %>"
+
+[var.pausers]
+pauser1 = "0x140d001689979ee77C2FB4c8d4B5F3E209135776"
+pauser2 = "0xA73d7b822Bfad43500a26aC38956dfEaBD3E066d"
+pauser3 = "0xf94e5Cdf41247E268d4847C30A0DC2893B33e85d"
+pauser4 = "0xC68ed61DCe11Ba16586bCa350139cFDFc65D1Ca6"
+
+[var.redstone_oracles]
+redstonePriceFeedUSDC = "0x31A43dB62eC5c07885A89d78551243FE8D2D44D5"
+redstonePriceFeedETH = "0xFBA40360559a1bcCC608d9cA07deDE8588460874"
+redstonePriceFeedBTC = "0xd03Dc3047a8FA1Bca82A8cCda7b389B851A87861"
+
+[var.socket_contracts]
+usdcController = "0x0000000000000000000000000000000000000001"
```

Comment:
comment out for now?

Reply:
not really, we give permissions to the ONE_ADDRESS which is safe (esp. it's testnet) - we just need to plugin the actual adresses when time.